### PR TITLE
Update with overcome-fear/quote.md file

### PR DESCRIPTION
**Pull request by Nidhi16**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1840)

### CODE REVIEW

**Comment by Nidhi16** [comment]
Please review this, @casey-collab

File: english/overcome-fear/quotes.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+OVERCOME YOUR FEAR BY READING FOLLOWING QUOTES:
```
Comment (casey-collab) [inline english/overcome-fear/quotes.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Nidhi16** [comment]
Okay, Thank you!

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
It looks like now the file exists in both places - you'll need to remove it from one of them. Also, the file is still not reachable. The problem is not what directory it's in - the problem is that none of the existing story has options that lead to this page.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**